virtualevent-demo/src/main/res/layout/fragment_overview.xml

```diff
@@ -1,22 +1,423 @@
+            android:text="@string/overview_description"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/description" />
+
+        <TextView
+            android:id="@+id/partners"
+            style="@style/OverviewHeader"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/margin_medium"
+            android:text="Partners"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/description_body" />
+
+        <GridLayout
+            android:id="@+id/partners_logos"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/margin_medium"
+            android:layout_marginVertical="12dp"
+            android:columnCount="2"
+            app:layout_constraintTop_toBottomOf="@+id/partners">
+
+            <androidx.appcompat.widget.AppCompatImageView
+                android:layout_width="0dp"
+                android:layout_columnWeight="1"
+                android:layout_margin="@dimen/margin_small"
+                android:src="@drawable/ic_red_canary_logo" />
+
+            <androidx.appcompat.widget.AppCompatImageView
+                android:layout_width="0dp"
+                android:layout_columnWeight="1"
+                android:layout_margin="@dimen/margin_small"
+                android:src="@drawable/ic_symantec_logo" />
+
+            <androidx.appcompat.widget.AppCompatImageView
+                android:layout_width="0dp"
+                android:layout_columnWeight="1"
+                android:layout_margin="@dimen/margin_small"
+                android:src="@drawable/ic_crowdstrike_logo" />
+
+            <androidx.appcompat.widget.AppCompatImageView
+                android:layout_width="0dp"
+                android:layout_columnWeight="1"
+                android:layout_margin="@dimen/margin_small"
+                android:src="@drawable/ic_cybereason_logo" />
+
+            <androidx.appcompat.widget.AppCompatImageView
+                android:layout_width="0dp"
+                android:layout_columnWeight="1"
+                android:layout_margin="8dp"
+                android:src="@drawable/ic_goguardian_logo" />
+
+            <androidx.appcompat.widget.AppCompatImageView
+                android:layout_width="0dp"
+                android:layout_columnWeight="1"
+                android:layout_margin="@dimen/margin_small"
+                android:src="@drawable/ic_logrythm_logo" />
+
+        </GridLayout>
+
+        <TextView
+            android:id="@+id/schedule"
+            style="@style/OverviewHeader"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_margin="18dp"
+            android:text="Schedule"
```

Comment:
Even in a sample app, I would still extract text strings

Reply:
done